docker/Dockerfile

```diff
@@ -72,18 +72,17 @@ ENV MINIFI_VERSION ${MINIFI_VERSION}
 RUN addgroup -g ${GID} ${USER} && adduser -u ${UID} -D -G ${USER} -g "" ${USER} && \
     install -d -o ${USER} -g ${USER} ${MINIFI_BASE_DIR}
 COPY --chown=${USER}:${USER} . ${MINIFI_BASE_DIR}
-RUN if [ -d "${MINIFI_BASE_DIR}/.ccache" ]; then mv ${MINIFI_BASE_DIR}/.ccache /home/${USER}/.ccache; fi
 
 USER ${USER}
 
-ENV PATH /usr/lib/ccache/bin:${PATH}
 RUN mkdir ${MINIFI_BASE_DIR}/build
 WORKDIR ${MINIFI_BASE_DIR}/build
-RUN cmake -DSTATIC_BUILD= -DSKIP_TESTS=${DOCKER_SKIP_TESTS} ${MINIFI_OPTIONS} -DAWS_ENABLE_UNITY_BUILD=OFF -DCMAKE_BUILD_TYPE="${CMAKE_BUILD_TYPE}" .. && \
+RUN export PATH=/usr/lib64/ccache/bin${PATH:+:${PATH}} && \
```

Comment:
what is the benefit to using `RUN export ...` instead of the `ENV` command?

Reply:
* I wasn't (and still amn't) sure if ENV does an export;
* the centos build does it this way, and I wanted both builds to do the same thing;
* we know that caching in the centos build works; I'm not sure if caching in the docker build was working before this change.

Reply:
I see, thanks!